frontend/model/database.js

```diff
@@ -360,7 +377,7 @@ sbp('sbp/selectors/register', {
 })
 
 // ======================================
-// Archve for proposals and anything else
+// Archive for proposals and anything else
```

Comment:
Also found bad string in `'gi.db/filesCache/delete'` error handler:

```js
// ... existing code ...
}).catch(e => {
  // Wrong message: says "load" instead of "delete"
  console.error('[gi.db/filesCache/load] Error updating keys')
})
// Should be:
}).catch(e => {
  console.error('[gi.db/filesCache/delete] Error updating keys:', e)
})
```

Reply:
There is no race condition. See `if (idx !== -1) {`. However, the bad string is an issue.

Reply:
If you're satisfied, please resolve this conversation. The reason there's no race condition, expanding further, is that the index and the value are updated, ideally, atomically (hence the queue).